include/drivers/console/bm_serial.h

```diff
@@ -16,29 +16,30 @@
 
 #include <stdlib.h>
 #include <stdint.h>
+#include <kernel.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-#define NUM_BM_FRAMES           20
-#define MAX_BM_FRAME_SIZE       255
+#define NUM_BM_FRAMES           40
```

Comment:
All of these defines should get converted to Kconfig options so they can be easily set via prj.conf